index.bs

```diff
@@ -651,7 +651,7 @@ that shares the same buffer as the input tensor. (In the case of reshape,
 the entire data is shared, while in the case of slice, a part of the input data is shared.)
 The implementation may use views, as above, for intermediate values.
 
-Before the execution, the computation graph that is used to compute one or more specified outputs needs to be compiled and optimized. The key purpose of the compilation step is to enable optimizations that span two or more operations, such as operation or loop fusion.
+Before the execution, the computation graph that is used to compute one or more specified outputs needs to be constructed, compiled, and optimized. The key purpose of the compilation step is to enable optimizations that span two or more operations, such as operation or loop fusion. The user agent may also perform these optimizations during graph construction.
```

Comment:
Does it make sense to use term "conversion"? That would refer to the procedure which converts the WebNN graph to native graph, rather than the "construction" of WebNN graph by JS code. This hopefully would align with the "converting" step of [`build()`](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-build) method

> Let graphImpl be the result of converting [this](https://webidl.spec.whatwg.org/#this)'s [graph](https://www.w3.org/TR/webnn/#mlgraphbuilder-graph) with operands, operators, inputs, and outputs’s [values](https://infra.spec.whatwg.org/#map-getting-the-values) into an [implementation-defined](https://infra.spec.whatwg.org/#implementation-defined) format which can be interpreted by the underlying platform.

I suppose the user agent could perform these optimizations at that step, like Chromium DirectML does.

```suggestion
Before the execution, the computation graph that is used to compute one or more specified outputs needs to be converted, compiled, and optimized. The key purpose of the compilation step is to enable optimizations that span two or more operations, such as operation or loop fusion. The user agent may also perform these optimizations during graph conversion.
```